### PR TITLE
Add tests for WGRUPCON default rate and zero guiderate potential fallback

### PR DESCRIPTION
**Pull request by vkip**

### CODE REVIEW

File: model2/include/PVT-DRY-GAS-LIVE-OIL.INC [hunk 1]
```diff
@@ -0,0 +1,338 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2018 Equinor
+
```
Comment (totto82) [inline model2/include/PVT-DRY-GAS-LIVE-OIL.INC:7]: 2018? Is this just a copy of some other PVT file?

**Review by totto82 — APPROVED** [review]
Thanks for the tests.

File: model2/include/PVT-DRY-GAS-LIVE-OIL.INC (hunk 1 above)
Comment (vkip) [reply model2/include/PVT-DRY-GAS-LIVE-OIL.INC:7]: Good point, I'll update. (Not just a copy, but a modified version of another file).
